src/osd/ReplicatedPG.cc

```diff
@@ -5892,6 +5894,24 @@ int ReplicatedPG::do_osd_ops(OpContext *ctx, vector<OSDOp>& ops)
       }
       break;
 
+    case CEPH_OSD_OP_ASSERT_INTERVAL:
+      ++ctx->num_read;
+      {
+	epoch_t epoch = op.assert_interval.epoch;
+	epoch_t pg_epoch = ctx->pg->get_epoch();
```

Comment:
could just use `get_epoch()` here. as `ctx->pg` is identical to `this`, i think.